<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorBloom</class>
 <widget class="QWidget" name="InspectorBloom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Bloom</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Intensity</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Treshhold</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Exposure Scale</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_pIntensityEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Tint</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Size</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="m_pTreshholdEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="m_pExposureScaleEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="m_pSizeEdit"/>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="m_pTintColorButton">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Button">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>0</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pTintPickButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Pick</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pTintColorButton</sender>
   <signal>clicked()</signal>
   <receiver>InspectorBloom</receiver>
   <slot>pickColorFromDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pTintPickButton</sender>
   <signal>clicked()</signal>
   <receiver>InspectorBloom</receiver>
   <slot>pickColorFromDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pIntensityEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorBloom</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pTreshholdEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorBloom</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pExposureScaleEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorBloom</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pSizeEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorBloom</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pickColorFromDialog()</slot>
  <slot>valueChanged()</slot>
 </slots>
</ui>
